.github/workflows/integration.yaml

```diff
@@ -31,9 +31,8 @@ jobs:
      name: Code linters
      runs-on: ubuntu-latest
      steps:
-       - uses: actions/checkout@v2
-       - name: install python
```

Comment:
@akx Why did you remove the name?

Reply:
Not all of the `actions/setup-python` steps in the YAML files had a name beforehand, anyway.

I see no actual value in a longer YAML file with the only visible change being "install python" being shown instead of "Run actions/setup-python@v4" in logs.